app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -79,169 +112,103 @@ form {
 
-  fieldset > ol > li {
-    fieldset {
-      position:relative;
-      padding: 0;
-      margin-bottom: 0;
-
-      legend {
-        position:absolute;
-        width:95%;
-        padding-top:0.1em;
-        left: 0px;
-        font-size: 100%;
-        font-weight: normal;
-        span { position:absolute; }
-        &.label label { position:absolute; }
-      }
-
-      &:not(.has_many_fields) ol {
-        float: left;
-        width: 74%;
-        margin: 0;
-        padding: 0 0 0 20%;
-
-        li {
-          padding: 0;
-          border: 0;
-        }
-      }
+  legend.label {
+    font-size: inherit;
+    left: 0;
+    padding-top: 0.1em;
+    position: absolute
+  }
 
-      &.has_many_fields ol {
-        float: left;
-        width: 100%;
-        margin: 0;
-      }
-    }
+  .choices-group,
+  .fragments-group {
+    float: left;
+    padding: 0 0 0 20%;
+    width: 74%;
   }
 
-  /* Text Fields */
-  input[type=text], input[type=password], input[type=email], input[type=number], input[type=url], input[type=tel], textarea {
-    width: calc(80% - #{$text-input-total-padding});
-    border: $border-width solid #c9d0d6;
-    @include rounded;
-    font-size: 0.95em;
-    @include sans-family;
-    outline: none;
-    padding: 8px $text-input-horizontal-padding 7px;
-
-    &:focus {
-      border: $border-width solid #99a2aa;
-      @include shadow(0,0,4px,#99a2aa);
-    }
+  .choice,
+  .fragment {
+    margin: 0 0 0.5em 0;
   }
 
-  fieldset > ol > li {
+  .fragment {
+    display: inline-block;
+    margin: 0 5px 0 0;
+    width: auto;
 
-    /* Hints */
-    p.inline-hints {
-      font-size: 0.95em;
-      font-style: italic;
-      color:#666;
-      margin: 0.5em 0 0 20%;
+    label {
+      display: none;
     }
+  }
 
-    /* Date and Time Fields */
-    &.date_select, &.time_select, &.datetime_select {
-      fieldset ol li {
-        float:left; width:auto; margin:0 0.5em 0 0;
-        label { display: none; }
-        input { display:inline; margin:0; padding:0;  }
-      }
-    }
+  // BOOLEAN OVERRIDES
 
-    /* Check Boxes or Radio fields */
-    &.check_boxes, &.radio {
-      fieldset ol {
-        margin-bottom:-0.6em;
-        li {
-          margin:0.1em 0 0.5em 0;
-          label {
-            float:none;
-            width:100%;
-            input { margin-right:0.2em; }
-          }
-        }
-      }
-    }
+  .boolean input,
+  .choice input {
+    margin: 0 0.3em 0 0;
+  }
 
-    /* Boolean Field */
-    &.boolean {
-      height: 1.1em;
-      label {
-        width: 80%;
-        padding-left:20%;
-        padding-right: 10px;
-        text-transform: none !important;
-        font-weight: normal;
-        input { margin:0 0.5em 0 0.2em; }
-      }
-    }
+  .boolean label {
+    padding-left:20%;
+    padding-right: 10px;
+    width: 80%;
+  }
 
-    /* Hidden fields */
-    &.hidden {
+  fieldset > ol > li {
+    fieldset {
+      position:relative;
```

Comment:
Properties should be ordered margin, padding, position<br>Colon after property should be followed by one space